app/views/school/home/show_la_funded_place.html.erb

```diff
@@ -0,0 +1,60 @@
+<%- title = t('page_titles.iss_home') %>
```

Comment:
this translation is not defined and therefore missing

Reply:
looks a review commit in haste has caused issues

Reply:
Good spot, that's because I accepted James change  in the comment to scope the `iss_` names

Reply:
Done